src/packaging/webots_distro.c

```diff
@@ -1125,7 +1134,7 @@ static void create_file(const char *name, int m) {
       }
 
 #ifdef WEBOTS_UBUNTU_16_04
-      fprintf(fd, "fakeroot dpkg-deb --build debian %s\n", distribution_path);
+      fprintf(fd, "fakeroot dpkg-deb -Zgzip --build debian %s\n", distribution_path);
```

Comment:
This improves the creation of the debian package from 11:53 minutes to 3:10 minutes